posthog/api/dashboard.py

```diff
@@ -210,6 +194,44 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> Dashboard:
 
         return dashboard
 
+    def _duplicate_tiles(self, dashboard: Dashboard, existing_tile: DashboardTile) -> None:
```

Comment:
This duplicates tiles, but I'd say the more important part is this duplicated _insights_, isn't that the point?
Arguably `_duplicate_insights` wouldn't be correct since text cards _are_ just tiles, but there's still an important distinction. So maybe `_deep_duplicate_tiles`, to indicate this is a "deep" copy (all the way down) instead of shallow (just the surface tiles)?